src/util/sienna/helper_functions.jl

```diff
@@ -151,9 +151,9 @@ function get_forecast_values(ts::Nothing)
 end
 
 function get_outage_time_series_data(
-    gen::Union{SI, BR},
+    gen::Union{PSY.StaticInjection, PSY.Branch},
```

Comment:
I'm curious why this change? Is Aqua pointing to an ambiguity here?

Reply:
Would this work on any subtype of PSY.Branch and PSY.StaticInjection?

Reply:
Aqua is pointing out an unused type variable. Any subtype of PSY.Branch and PSY.StaticInjection should satisfy `T <: Union{PSY.StaticInjection, PSY.Branch}`